server/api/meeting/index.post.ts

```diff
@@ -19,32 +43,51 @@ export default defineEventHandler(async (event) => {
 		const currentTime = new Date()
 
 		// Check if meeting start time is in the past
-		if (startTime < currentTime) {
-			throw createError({
-				statusCode: 400,
-				statusMessage: 'Cannot schedule a meeting in the past.',
-			})
-		}
+		if (startTime < currentTime)
+			throw createError({ statusCode: 400, statusMessage: 'Cannot schedule a meeting in the past.' })
+		if (endTime < startTime)
+			throw createError({ statusCode: 400, statusMessage: 'The meeting end time must be after the start time.' })
 
-		// Check if meeting end time is greater than start time
-		if (endTime < startTime) {
-			throw createError({
-				statusCode: 400,
-				statusMessage: 'The meeting end time must be after the start time.',
-			})
-		}
+		// Create room in 100ms
+		const roomResponse = await fetch('https://api.100ms.live/v2/rooms', {
```

Comment:
Wondering if we should wrap this in a try catch in case there is an error?

Reply:
It's already wrapped in a try catch, but I'm not sure if an HMS error will be handled correctly.

Reply:
Should handle the hms errors now